docs/conf.py

```diff
@@ -59,7 +59,9 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns += ['_templates', 'notebooks/README.rst']
+exclude_patterns += ['_templates',
+                     'notebooks/README.rst',
+                     'notebooks/Debug']
```

Comment:
If https://github.com/StingraySoftware/notebooks/pull/57 is merged as-is, then yes, this will also need to go to `main` here. If the `Debug` directory is removed from that PR, then we can remove it here too. But I assume it's wanted...

Reply:
The Debug directory, as I am using it, should contain all tests we've done to assess that some new code worked as expected, or that it works better than the existing code it replaces. Something more for us than for the general user, in fact it is not included in the docs.